tasks/copy_to_glacier/copy_to_glacier.py

```diff
@@ -95,51 +135,103 @@ def task(event: Dict[str, Union[List[str], Dict]], context: object) -> Dict[str,
-        print(f'{CONFIG_MULTIPART_CHUNKSIZE_MB_KEY} is not set for config. '
-              f'Using default value of {multipart_chunksize_mb}.')
+        multipart_chunksize_mb = int(os.environ["DEFAULT_MULTIPART_CHUNKSIZE_MB"])
+        LOGGER.debug(
+            "{CONFIG_MULTIPART_CHUNKSIZE_MB_KEY} is not set for config. Using default value of {multipart_chunksize_mb}.",
+            CONFIG_MULTIPART_CHUNKSIZE_MB_KEY=CONFIG_MULTIPART_CHUNKSIZE_MB_KEY,
+            multipart_chunksize_mb=multipart_chunksize_mb,
+        )
     else:
         multipart_chunksize_mb = int(multipart_chunksize_mb_str)
 
+    try:
+        metadata_queue_url = os.environ.get("METADATA_DB_QUEUE_URL")
+        if metadata_queue_url is None or len(metadata_queue_url) == 0:
+            raise KeyError("METADATA_DB_QUEUE_URL environment variable is not set.")
+    except KeyError:
+        LOGGER.error("METADATA_DB_QUEUE_URL environment variable is not set.")
+        raise
+
     granule_data = {}
     copied_file_urls = []
 
+    # initiate empty SQS body dict
+    sqs_body = {}
+    sqs_body["provider"] = {}
+    sqs_body["collection"] = {}
+    sqs_body["granule"] = {}
+    # 'providerName' set to None because Cumulus only returns providerId for now.
+    # In case it is available in the future, update orca_copy_to_glacier_workflow.asl.json and config.json as needed
+    sqs_body["provider"]["providerName"] = None
+    sqs_body["provider"]["providerId"] = config["providerId"]
+    sqs_body["collection"]["shortname"] = config["collectionShortname"]
+    sqs_body["collection"]["version"] = config["collectionVersion"]
+    # Cumulus currently creates collectionId by concating shortname + ___ + version 
+    # See https://github.com/nasa/cumulus-dashboard/blob/18a278ee5a1ac5181ec035b3df0665ef5acadcb0/app/src/js/utils/format.js#L342
+    sqs_body["collection"]["collectionId"] = (
+        config["collectionShortname"] + "___" + config["collectionVersion"]
+    )
     # Iterate through the input granules (>= 0 granules expected)
     for granule in granules_list:
         # noinspection PyPep8Naming
-        granuleId = granule['granuleId']
+        granuleId = granule["granuleId"]
         if granuleId not in granule_data.keys():
-            granule_data[granuleId] = {'granuleId': granuleId, 'files': []}
+            granule_data[granuleId] = {"granuleId": granuleId, "files": []}
+        # populate the SQS body for granules
+        sqs_body["granule"]["cumulusGranuleId"] = granuleId
+        granule["createdAt"] = "2021-10-08T19:24:07.605323Z" # remove this line once this card is complete https://bugs.earthdata.nasa.gov/browse/CUMULUS-2718
+        sqs_body["granule"]["cumulusCreateTime"] = granule["createdAt"].replace("Z", "+00:00")
+        sqs_body["granule"]["executionId"] = config["executionId"]
+        sqs_body["granule"]["ingestTime"] = datetime.now(timezone.utc).isoformat()
+        sqs_body["granule"]["lastUpdate"] = datetime.now(timezone.utc).isoformat()
```

Comment:
You could set this to ingestTime if you wanted.

Reply:
lets keep it as lastUpdate since there is already one ingestTime